express/src/views/sign-in/resend-text-code.njk

```diff
@@ -6,7 +6,7 @@
 {% block afterForm %}
   <p class="govuk-body">
     If you no longer have access to the phone number linked to your account, you can
-    <a class="govuk-link" href="/test/security-check-change-number">change your phone number</a>.
+    <a class="govuk-link" href="/security-check-change-number">change your phone number</a>.
```

Comment:
This route is in the testing routes

Reply:
I will remove the code, as it is related to phone update during sign-in, which is not needed at the moment.